funflow.cabal

```diff
@@ -72,6 +72,7 @@ Library
                , process
                , random
                , stm
+               , sqlite-simple
```

Comment:
No, [`sqlite-simple`](http://hackage.haskell.org/package/sqlite-simple) depends on [`direct-sqlite`](http://hackage.haskell.org/package/direct-sqlite) which contains a sqlite implementation in C. So, it only requires a C compiler to build. Optionally, via cabal flag `systemlib`, it can link against a preinstalled sqlite library, instead.

Reply:
Gotcha, sounds good.